internal/mapping.go

```diff
@@ -30,23 +30,29 @@ import (
 // a) the new table name is legal
 // b) the new table name doesn't clash with other Spanner table names
 // c) we consistently return the same name for this table.
+//
+// conv.UsedNames tracks Spanner names that have been used for table names, foreign key constraints
+// and indexes. We use this to ensure we generate unique names when
+// we map from MySQL to Spanner since Spanner requires all these names to be
```

Comment:
Change from "we map from MySQL ... " to "we map from source databases .... not differ only in case" This behaviour is not enforced in all source databases, an example of such is Mysql"